catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliFilterFactory.java

```diff
@@ -410,7 +413,7 @@ public String buildBeyondFilter(String propertyName, String wkt, double distance
     }
 
     public String convertWktToBqs(String wkt) {
-        WKTReader wktReader = new WKTReader();
+        WKTReader wktReader = new WKTReader(GEOMETRY_FACTORY);
```

Comment:
This WKTReader is potentially not thread safe.  I recommend using thread local to avoid any issues